Tests/Integration/PauseWithPredisTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Resque\Plugins\Tests\Integration;
+
+use Resque;
+use Predis\Client;
+
+/**
+ * Pause tests.
+ *
+ * @package     PHP Resque Pause
+ * @author      Wedy Chainy <wedy.chainy@bigcommerce.com>
```

Comment:
I haven't modified the author at all. I could put tools if you like.
On Jun 21, 2015 4:24 PM, "Cameron Walsh" <notifications@github.com> wrote:

> In Tests/Integration/PauseWithPredisTest.php
> <https://github.com/bigcommerce/php-resque-pause/pull/8#discussion_r32899247>
> :
>
> > @@ -0,0 +1,51 @@
> > +<?php
> > +namespace Resque\Plugins\Tests\Integration;
> > +
> > +use Resque;
> > +use Predis\Client;
> > +
> > +/**
> > + * Pause tests.
> > + *
> > + * @package     PHP Resque Pause
> > + * @author      Wedy Chainy <wedy.chainy@bigcommerce.com>
>
> Really?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bigcommerce/php-resque-pause/pull/8/files#r32899247>.
>